scripts/generate_llms.sh

```
@@ -0,0 +1,714 @@
+
+# Parse args: PLATFORM as first positional, optional REPO_URL, simple flags
+PRINT_STDOUT="false"
+USE_LLM="false"
+LLM_FORCE_ALL="false"
+OPENAI_BASE_URL="${OPENAI_BASE_URL:-https://api.openai.com}"
+OPENAI_MODEL="${OPENAI_MODEL:-gpt-5-mini}"
+LLM_SLEEP_SEC="${LLM_SLEEP_SEC:-0.25}"
+LLM_SOURCE="${LLM_SOURCE:-raw}" # raw | local
+LLM_LOG="${LLM_LOG:-true}"      # log LLM outputs to stderr when true
+LLM_MAX_BYTES="${LLM_MAX_BYTES:-0}" # 0 = no limit; otherwise truncate to this many bytes
+args=("$@")
```

Shellcheck (0.11.0)

[warning] args appears unused. Verify use (or export if used externally).

(SC2034)